src/Protocol/Diaspora.php

```diff
@@ -2796,7 +2795,7 @@ private static function receiveRetraction(array $importer, $sender, stdClass $da
 	 *
 	 * @return int The message id of the newly created item
 	 */
-	private static function receiveStatusMessage(array $importer, stdClass $data, $xml)
+	private static function receiveStatusMessage(array $importer, $data, $xml)
```

Comment:
Then why not set `SimpleXMLElement` here?

Reply:
This had been some quick fix between waking up and getting under the shower. There hadn't been much time for making it nice.

Reply:
Of course you can fix it - but I really recommend letting the code run for some time on your machine and checking the error logs before submitting the PR.